contracts/Bridge.sol

```diff
@@ -8,7 +8,7 @@ pragma solidity 0.6.11;
 // import "./IVoteManager.sol";
 // import "./JobStorage.sol";
 // import "./IStateManager.sol";
-// import "openzeppelin-solidity/contracts/math/SafeMath.sol";
```

Comment:
Since we won't be needing `SafeMath` (at v0.8.0+), let's remove this import @adi44

Reply:
removed this  import line @dev1644